src/infrastructure/http/deps.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Request
+
+from apps.shared.errors import ForbiddenError
+from infrastructure.http.domain import MindLoggerHeaders
+
+
+async def mindlogger_headers(request: Request) -> MindLoggerHeaders:
+    try:
+        mindlogger_content_source = MindLoggerHeaders(
+            mindlogger_content_source=request.headers.get(
+                "mindlogger-content-source"
```

Comment:
add `web` as default, because if does not exists, it will be considered as `web` not `admin`

Reply:
It is not aligned yet as far as I know

Reply:
Anyway. Changed